### ISSUE TITLE
Keep track of results

### ISSUE THREAD

**Issue by JoshVarty** [JoshVarty/KaggleClouds#5]
### 10 epochs 30 epochs
| Description | n_train | NFolds | Threshold/MinSize | Valid | LB |
| :---         |:---         |      :---:      |    :---:      |          :---: |:---: |
| Resnet18, get_transforms() |  | 2 |  0.5/10000   | 0.446 | 0.600|
| sample_submission.csv     | - | - |  -     |  | 0.000 |
| empty_submission.csv     | - |- |  -     |  | 0.477 |
| Resnet18, no warp, no rotate  | 4000 |2 |  0.5/10000   | 0.452  | 0.597 |
| XResnet18, no warp, no rotate (no pretrain)  | 4000 |2 |  0.5/10000   | 0.401  | 0.588 |
| Resnet18, get_transforms() DiceLoss  | 4000 | 2 |  0.5/10000  | 0.508 | 0.607 |
| Resnet18, no warp, no rotate DiceLoss  | 4000 | 2 |  0.5/10000   | 0.490  | 0.600 |

### 10 epochs 60 epochs
| Description | n_train | NFolds | Threshold/MinSize | Valid | LB |
| :---         |:---         |      :---:      |    :---:      |          :---: |:---: |
| Resnet18, get_transforms() DiceLoss  | 4000 | 2 |  0.5/10000  | 0.486 | 0.601 |
| Resnet18, get_transforms() DiceLoss mixup  | 4000 | 2 |  0.5/10000  | 0.486 | 0.615 |

### 10 epochs 90 epochs
| Description | n_train | NFolds | Threshold/MinSize | Valid | LB |
| :---         |:---         |      :---:      |    :---:      |          :---: |:---: |
| Resnet18, get_transforms() DiceLoss mixup  | 4000 | 2 |  0.5/10000  | 0.497 | 0.607 |

**Comment by JoshVarty** [comment]
Repeat experiments with new validation set and thresholded dice score:

### 10 epochs 30 epochs
| Description | n_train | NFolds | Threshold/MinSize | Valid | LB |
| :---         |:---         |      :---:      |    :---:      |          :---: |:---: |
| Resnet18, get_transforms() |  | 1 |  0.5/10000   | 0.5069 | 0.602  |
| XResnet18, get_transforms()  | - | 1 |  0.5/10000   | 0.5032 | 0.627 |
| Resnet18, get_transforms() DiceLoss  | - | 1|  0.5/10000  | 0.5322 | 0.619 |


My validation approach doesn't appear to be any better. :cry:

**Comment by JoshVarty** [comment]
Our approach doesn't seem to be working very well. We'll start to look other approaches and try to improve upon those.

### Forked Notebook
| Description | Threshold/MinSize | Valid | LB |
| :---      |    :---:      |    :---: |:---: |
| Resnet50  | optimized | - | 0.640 |
| Resnet50  | 0.5/10000 | - | 0.634 |
| Resnet50/RAM/Our Resize | optimized | - | 0.634 |
| Resnet50/RAM/(320,640) | optimized | - | 0.634 |
| Resnet50/RAM/(320,640)/Their Masks | optimized |  |  |
